catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -212,6 +212,11 @@
             <artifactId>mime-core-impl</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>ddf.catalog.plugin</groupId>
+            <artifactId>catalog-plugin-metacard-validation</artifactId>
+            <version>2.9.0-SNAPSHOT</version>
```

Comment:
Thanks, I forgot to check after automatically adding the dependency. This is actually removed in my most recent version.